app/components/shared/event-item.tsx

```diff
@@ -47,9 +62,39 @@ export function EventItem({ event }: { event: Event }) {
         </div>
 
         <p className="text-sm text-muted-foreground">
-          <time>{formatPublished(event.dateTimeStart)}</time>
+          <time>
+            <span>{formatPublished(event.dateTimeStart)}</span>
+            <br />
+            <span className="text-muted-foreground">
+              {formatTime(event.dateTimeStart)} –{" "}
+              {formatPublished(event.dateTimeStart) !==
+              formatPublished(event.dateTimeEnd)
+                ? formatPublishedWithTime(event.dateTimeEnd)
+                : formatTime(event.dateTimeEnd)}{" "}
+              WIB
+            </span>
+          </time>
         </p>
 
+        {event.category?.symbol === "HYBRID" ? (
+          <p className="flex flex-row items-center gap-1">
+            <Iconify className="text-2xl" icon="ph:map-pin" />{" "}
+            {`${event.location?.label} / ${event.media?.name}`}
+          </p>
+        ) : event.category?.symbol === "IN_PERSON" ? (
+          <p className="flex flex-row items-center gap-1">
+            <Iconify className="text-2xl" icon="ph:map-pin" />{" "}
+            {event.location?.label}
+          </p>
+        ) : event.category?.symbol === "ONLINE" ? (
+          <p className="flex flex-row items-center gap-1">
+            <Iconify className="text-2xl" icon="ph:map-pin" />{" "}
+            {event.media?.name}{" "}
+          </p>
+        ) : (
+          <div />
+        )}
```

Comment:
@fikrialwan Better to avoid ternary and some variables to be easier to scan.

Use like:

```ts
isHybrid = event.category?.symbol === "HYBRID
```

```tsx
isHybrid && <p>...</p>
```

Reply:
I've use some variable for readable code instead of use ternary on this commit https://github.com/bandungdevcom/bandungdev.com/pull/42/commits/ff3311e5774d960e8dfdcd3911b5e6b8a0b384c8